<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extrem Data Base For Video Game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/icon/ressources/icone.png</normaloff>:/icon/ressources/icone.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>551</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QComboBox" name="comboConsole">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Console :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Titre :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineTitle">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Année :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Développeurs :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="liveDev">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAdd">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>50</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableGames">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAjouter_console"/>
    <addaction name="actionSupprimer_ligne"/>
    <addaction name="actionEditer_ligne"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExporter_en_csv"/>
    <addaction name="actionImporter_un_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAjouter_console">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icon/ressources/console 64x64.png</normaloff>:/icon/ressources/console 64x64.png</iconset>
   </property>
   <property name="text">
    <string>Ajouter console</string>
   </property>
  </action>
  <action name="actionSupprimer_ligne">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icon/iSuppr</normaloff>:/icon/iSuppr</iconset>
   </property>
   <property name="text">
    <string>Supprimer ligne</string>
   </property>
  </action>
  <action name="actionEditer_ligne">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icon/iEdit</normaloff>:/icon/iEdit</iconset>
   </property>
   <property name="text">
    <string>Editer ligne</string>
   </property>
  </action>
  <action name="actionExporter_en_csv">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icon/ressources/export32x32.png</normaloff>:/icon/ressources/export32x32.png</iconset>
   </property>
   <property name="text">
    <string>Exporter un CSV</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionImporter_un_CSV">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icon/ressources/import32x32.png</normaloff>:/icon/ressources/import32x32.png</iconset>
   </property>
   <property name="text">
    <string>Importer un CSV</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
